src/element-style-observer.js

```diff
@@ -5,13 +5,14 @@ import { toArray, wait, getTimesFor } from "./util.js";
 import RenderedObserver from "./rendered-observer.js";
 
 // Start bug detection early if we have a DOM
+let allowDiscrete = "";
 if (globalThis.document) {
-	Bugs.detect();
-}
+	Bugs.detectAll();
```

Comment:
```suggestion
	bugs.detectAll();
```

Reply:
We also need to fix import on line 1:

`import { bugs } from "./util/detect-bugs.js";`